src/components/share-results.tsx

```diff
@@ -17,19 +17,7 @@ const ShareContainer = styled.div`
   }
 `
 
-const ShareButton = styled.a.attrs({ target: '_blank' })`
-  font-size: ${props => props.theme.sizes.buttonText};
-  font-family: ${props => props.theme.fontFamily};
-  font-weight: 500;
-  border-radius: 28px;
-  padding: calc(${props => props.theme.sizes.buttonText} * 0.75) 24px;
-  box-shadow: 0 5px 20px rgba(0, 0, 0, 0.1);
-  min-width: 6em;
-  margin: 6px;
-  color: #fff;
-  text-decoration: none;
-  display: block;
-  text-align: center;
+const ShareButton = styled(CtaButton)`
```

Comment:
if you add `withComponent('a')` at end of style for ShareButton, then you won't need to use the polymorphic `as = "a"`